x/uibc/module/abci.go

```diff
@@ -14,7 +14,7 @@ func BeginBlock(ctx sdk.Context, keeper keeper.Keeper) {
 
 	// reset quotas
 	if quotaExpires == nil || quotaExpires.Before(ctx.BlockTime()) {
-		util.Panic(keeper.ResetQuota(ctx))
+		util.Panic(keeper.ResetAllQuotas(ctx))
```

Comment:
May be can you rename to `ResetAllOutflows` instead of `ResetAllQuotas`

Reply:
Actually, here `ResetAllQuotas` makes more sense. Outflow is an aggregate.